react-modal/my-app/src/App.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+
+import './App.css';
+import { Modal } from './Modal';
+
+function App() {
+  // const modal = useRef<HTMLDialogElement>(null);
+  const [open, setOpen] = useState(false);
+
+  return (
+    <>
+      <button
+        onClick={() => {
+          setOpen(true);
+        }}>
+        Delete Me!
+      </button>
+
+      <Modal
+        children={
```

Comment:
You can pass in the children this way, but it looks much more natural, and like HTML, if you just put them as nested components:
```
<Modal ...>
  <div className="flex flex-col">
  ...
  </div>
</Modal>
```

When React sees nested components, it gathers them up and passes them to the parent in the `children` prop. That way you don't need to explicitly use the children prop.